extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcRecorder.java

```diff
@@ -171,6 +171,14 @@ public void handle(AsyncResult<OAuth2Auth> event) {
                 });
 
                 auth = cf.join();
+
+                String endSessionEndpoint = OAuth2AuthProviderImpl.class.cast(auth).getConfig().getLogoutPath();
```

Comment:
Should we just call it `logoutEndpoint` :-) ?

Reply:
I don't really care. This is just the name of the claim from the discovery document.